smart-contracts/contracts-common/concordium-contracts-common/src/schema.rs

```diff
@@ -1215,6 +1217,21 @@ mod impls {
             Ok(param_schema)
         }
 
+        // Returns an event schema from a versioned module schema
+        pub fn get_event_schema(&self, contract_name: &str) -> Result<Type, VersionedSchemaError> {
+            let versioned_contract_schema = get_versioned_contract_schema(self, contract_name)?;
+
+            let param_event = match versioned_contract_schema {
+                VersionedContractSchema::V0(_) => Err(VersionedSchemaError::ErrorNotSupported)?,
```

Comment:
The error message of `ErrorNotSupported` does not match this case, since it is about error information in the schema. So you should introduce another error for contract event
```suggestion
                VersionedContractSchema::V0(_) => Err(VersionedSchemaError::EventNotSupported)?,
```

Reply:
See comment above.